shared/src/commonTest/kotlin/co/nimblehq/mark/kmmic/domain/usecase/GetSurveysUseCaseTest.kt

```diff
@@ -0,0 +1,56 @@
+package co.nimblehq.mark.kmmic.domain.usecase
+
+import co.nimblehq.mark.kmmic.data.service.survey.model.SurveyApiModel
+import co.nimblehq.mark.kmmic.domain.model.Survey
+import co.nimblehq.mark.kmmic.domain.repository.SurveyRepository
+import co.nimblehq.mark.kmmic.dummy.dummy
+import kotlin.test.BeforeTest
+import io.kotest.matchers.shouldBe
+import io.mockative.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flowOf
+import org.koin.core.context.startKoin
+import org.koin.dsl.module
+import kotlin.test.Test
+import kotlinx.coroutines.test.runTest
+import org.koin.core.context.stopKoin
+import kotlin.test.AfterTest
+
+@ExperimentalCoroutinesApi
+class GetSurveysUseCaseTest {
+
+    @Mock
+    private val mockSurveyRepository = mock(classOf<SurveyRepository>())
+    private lateinit var useCase: GetSurveysUseCase
```

Comment:
Same here. Keeping the SUT separate makes it more readable in terms of code organization. Also, how about renaming it to `getSurveysUseCase` to follow the naming convention?

Reply:
@AVI5HEK A simple name `useCase` here is enough to know this is `getSurveysUseCase` cause we are in `GetSurveysUseCaseTest`, IMO 🙏